src/io/files.js

```diff
@@ -897,8 +918,12 @@ p5.prototype.httpDo = function () {
         type = 'json';
       } else if (path.indexOf('xml') !== -1) {
         type = 'xml';
-      } else {
+      } else if (path.indexOf('text') !== -1) {
```

Comment:
Thanks for this feedback! I was definitely way way to heavy handed with the scope of my edits, I've been looking at the set you identified in an earlier message and reverted to the current official p5 build of `files.js` as I start another attempt. I hope I'm not wasting to much of your time with this! Thank you again for your patience and feedback